src/main/resources/application.properties

```diff
@@ -1 +1 @@
-
+api.url = "https://api.infusionsoft.com/crm/rest/v1/%s"
```

Comment:
Was this used? It was also hard coded above.

Reply:
Was going to use this to make it configurable (addressing this https://github.com/amaceing/contact-tasks/pull/1#discussion_r295936618) but was running into some issues due to not being familiar with Spring Configuration stuff. I'll spend some more time on it and see if I can get it working. 

If I can the intention was to not have to use ReflectionTestUtils in the test file to change this to `localhost` for unit tests to hit the mock server. That's why both src/main/resources/application.properties and src/test/resources/application.properties. was going to change the URL through these files.